jenkins/swift-functional-tests/10-install-ring.sh

```diff
@@ -1,12 +1,28 @@
 #!/bin/bash -xue
 
+function get_AllowEncodedSlashes {
```

Comment:
The idea is to have the same commit for 0.3 than for master. 

Anyway it's required because otherwise AllowEncodedSlashes defaults to Off (https://github.com/scality/openstack-ci-scripts/blob/master/jenkins/ring-install.sh#L38) which forbid any encoded '/' in the path (the func tests actually have a lot of objects which have '/' in their name)